components/styles.js

```diff
@@ -49,6 +49,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     padding: 5,
     width: DEVICE_WIDTH / 7,
+    height: DEVICE_WIDTH / 7,
```

Comment:
Sorry for being non-responsive for quite some time, it's been quite busy. I understand the advantage of using flex here, but the way I am trying to fix this is by calculating the potential height of the calendar and in order to do that, I need to know the height of each row. If you think it's best to keep the flex here, the only way I can think of that could work is taking into account all the paddings and margins of each parts of the week-row to sum up to a row height, which seems a bit ugly and hard to maintain code. What do you think of this approach?